pages/services.md

```diff
@@ -38,9 +38,9 @@ AB Tech provides sound and lighting services to organizations affiliated with or
 <div class="card mb-5 border-0">
   <div class="row g-0">
     <div class="col-lg-7">
-      <div class="card-body p-0 ps-lg-3 mb-3 mb-lg-0 p-2">
+      <div class="card-body p-4 ps-lg-3 mb-3 mb-lg-0">
         <h3 class="card-title">Lighting</h3>
-        <p class="card-text">We can provide lighting on the ground for small shows or setup spans of overhead lighting for larger events. We can provide profiles, spots, washes, uplighting, and effect lighting. We have traditional neutral and colored lighting as well as LEDs, moving lights, and strobes. We also have theatrical-grade haze and fog.</p>
+        <p class="card-text">You've probably seen our lighting, be it the inside of the Orientation Tent, lights illuminating the walls of the University Center, or the lighting behind KPDC and DS. We can provide lighting for almost any need imaginable, whether it's mood lighting for your club social or pyrotechnics and an industry-grade lighting rig for the next AB Concert.</p>
```

Comment:
Remember that not just students (or even students involved with student organizations at all) read this--at the very least you may want to expand the KPDC, DS, and AB acronyms. You may also want to move the list of capabilities to another section or paragraph instead of completely removing.

Reply:
Tried balancing being technical with being approachable, lmk what you think about the last commit.

Reply:
Tried balancing being technical with being approachable and expanded acronyms, lmk what you think about the last commit.

Reply:
Tried balancing being technical with being approachable and removed the second part, lmk what you think about the last commit.

Reply:
Yeah, looks good! I made one small change (list items all start with "the," and an extra comma).